src/sentry/static/sentry/app/components/group/tagDistributionMeter.jsx

```diff
@@ -99,19 +88,16 @@ const TagDistributionMeter = createReactClass({
    */
 
   renderSegments() {
-    let data = this.state.data;
-    let totalValues = data.totalValues;
-
-    let totalVisible = data.topValues.reduce((sum, value) => sum + value.count, 0);
+    let {orgId, projectId, group, totalValues, topValues, tag} = this.props;
```

Comment:
`topValues` and `totalValues` need to be described in `propTypes` here (linter should catch this)

Reply:
done